pkg/manager/impl/util/single_task_execution.go

```diff
@@ -31,6 +31,8 @@ var defaultTimeout = ptypes.DurationProto(24 * time.Hour)
 
 const systemNamePrefix = ".flytegen.%s"
 
+const noInputNodeID = ""
```

Comment:
Propeller will know to fill this in with the start node?

Reply:
yes, confirmed with @EngHabu offline

Reply:
ohh got it